creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -763,30 +764,17 @@ class SnapbackSM {
            * If either secondary is in `unhealthyPeers` list, add it to `unhealthyReplicas` list
            */
           for (const secondary of secondaries) {
```

Comment:
seems like every code change in this outside of the below chunk is test code that should be reverted